tests/unit/VCRStorageFileTest.php

```diff
@@ -14,10 +14,12 @@ final class VCRStorageFileTest extends TestCase
 	public function setup(): void
 	{
 		parent::setUp();
-		$authHeader = ['Authorization' => 'Bearer '.$_ENV['SERVICE_ROLE']];
+		$dotenv = \Dotenv\Dotenv::createUnsafeImmutable(dirname(__DIR__, 2), '.env.test');
```

Comment:
We should use a path here e.g. __DIR__ . '/../../env.test'; vs dirname()

Reply:
Done